### PR TITLE
Milestone 2: Translate.py

### PR DESCRIPTION
**Pull request by RayceRossum**
Addresses Issue #14

### CODE REVIEW

File: AutoTransformPy/translate.py [hunk 1]
```diff
@@ -18,3 +24,43 @@ def translate (image_path, num_images, max_translation):
...
+    if not isinstance(max_translation, int):
+        raise TypeError("The maximum translation of images must be an integer.")
+
+    # Check for valid input paramter values
+    if num_images < 1:
+        raise ValueError("The number of images returned must be 1 or greater.")
+
+    org_image = imread(image_path)
+
+    print(org_image.shape)
```
Comment (everittB) [inline AutoTransformPy/translate.py]: Is this print statement supposed to still be here?

**Comment by alyciakb** [comment]
- Add new dependencies to the README `matplotlib`

**Comment by RayceRossum** [comment]
Matplotlib is commented and only used for development/testing. Might be better just to remove the debugging lines, Cool?

**Comment by alyciakb** [comment]
Okay, that's no problem then!

**Review by alyciakb — APPROVED** [review]
Looks good!

**Pull request merged**